src/test/java/core/basesyntax/reader/CsvFileReaderTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.reader;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exceptions.FileWasNotReadException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CsvFileReaderTest {
+    private static final int ZERO_INDEX_POSITION = 0;
+    private static final String TYPE_B = "b";
+    private static final String FRUIT_TYPE_BANANA = "banana";
+    private static final String QUANTITY_EXAMPLE = "20";
+    private static final String FILE_TO_READ = "src/test/resources/db/ReadTransactionTest";
+    private static final String EMPTY_FILE = "src/test/resources/db/empty.csv";
+    private static final String NOT_EXISTENT_FILE = "";
+
+    private CsvFileReader fileReader;
+
+    @BeforeEach
+    void setUp() {
+        fileReader = new CsvFileReader(FILE_TO_READ);
+    }
+
+    @Test
+    void reading_notNullFile_Ok() {
+        List<List<String>> actualLists = fileReader.readTransactions();
+
+        Assertions.assertNotNull(actualLists);
+    }
+
+    @Test
+    void reading_notEmptyFile_Ok() {
+        CsvFileReader fileReader = new CsvFileReader(FILE_TO_READ);
+        List<List<String>> actualLists = fileReader.readTransactions();
+
+        List<List<String>> expectedLists = new ArrayList<>();
+        expectedLists.add(List.of(
+                TYPE_B,
+                FRUIT_TYPE_BANANA,
+                QUANTITY_EXAMPLE)
```

Comment:
List of lists looks not good, you can use String and and check it using contains method